src/pages/Claim/Claim.component.tsx

```diff
@@ -25,7 +25,7 @@ export const ClaimComponent = ({ onSubmit }: ClaimComponentProps) => {
       <Box
         sx={{
           p: 0,
-          maxWidth: { xs: 1300 },
+          maxWidth: { xs: 1400 },
```

Comment:
btw, do we need to use xs?
```suggestion
          maxWidth: 1400,
```
shouldn't it be enough?